{"url": "https://api.github.com/repos/pytorch/pytorch/issues/comments/429476651", "html_url": "https://github.com/pytorch/pytorch/pull/12612#issuecomment-429476651", "issue_url": "https://api.github.com/repos/pytorch/pytorch/issues/12612", "id": 429476651, "node_id": "MDEyOklzc3VlQ29tbWVudDQyOTQ3NjY1MQ==", "user": {"login": "hartb", "id": 18429659, "node_id": "MDQ6VXNlcjE4NDI5NjU5", "avatar_url": "https://avatars1.githubusercontent.com/u/18429659?v=4", "gravatar_id": "", "url": "https://api.github.com/users/hartb", "html_url": "https://github.com/hartb", "followers_url": "https://api.github.com/users/hartb/followers", "following_url": "https://api.github.com/users/hartb/following{/other_user}", "gists_url": "https://api.github.com/users/hartb/gists{/gist_id}", "starred_url": "https://api.github.com/users/hartb/starred{/owner}{/repo}", "subscriptions_url": "https://api.github.com/users/hartb/subscriptions", "organizations_url": "https://api.github.com/users/hartb/orgs", "repos_url": "https://api.github.com/users/hartb/repos", "events_url": "https://api.github.com/users/hartb/events{/privacy}", "received_events_url": "https://api.github.com/users/hartb/received_events", "type": "User", "site_admin": false}, "created_at": "2018-10-12T22:10:10Z", "updated_at": "2018-10-12T22:10:10Z", "author_association": "CONTRIBUTOR", "body_html": "<p>The remaining verification failure is a flake8 complaint about code not touched by the PR.</p>", "body_text": "The remaining verification failure is a flake8 complaint about code not touched by the PR.", "body": "The remaining verification failure is a flake8 complaint about code not touched by the PR."}